packages/core/utils/src/generateBuildMetrics.js

```diff
@@ -85,10 +86,16 @@ async function getSourcemapSizes(
     }
 
     let sizeMap = new Map();
+    for (let source of sources) {
+      sizeMap.set(source, 0);
```

Comment:
Why is this needed? `let sourceSizes = new Array(sources.length).fill(0)` should handle this already?

Reply:
Ow yeah...